runtime/dev/src/lib.rs

```diff
@@ -508,15 +406,6 @@ impl pallet_saft_registry::Config for Runtime {
     type WeightInfo = weights::pallet_saft_registry::WeightInfo<Runtime>;
 }
 
-parameter_types! {
-    pub const ProposalSubmissionPeriod: <Runtime as frame_system::Config>::BlockNumber = 10;
-    pub const VotingPeriod: <Runtime as frame_system::Config>::BlockNumber = 10;
-}
```

Comment:
I'd like to make this network dependent, so that we can for example use different thresholds for the dev runtime